controllers/messenger.js

```diff
@@ -170,13 +170,13 @@ module.exports = () => {
     .catch(res.logger.error);
   }
 
-  function sendFeedback(res, senderId, intent, message) {
+  function sendFeedback(res, senderId, intent, rawMessage) {
+    let message = rawMessage.length >= config.maxMessageLength ? config.maxMessageLengthString : rawMessage;
```

Comment:
Warning: the part with `let message = rawMessage;` is "by ref", right? If you modify its content, it will be modified in the ref object.